internal/provider/resource_vault_cluster_test.go

```diff
@@ -10,23 +10,39 @@ import (
 	"github.com/hashicorp/terraform-provider-hcp/internal/clients"
 )
 
-var testAccVaultClusterConfig = `
-resource "hcp_hvn" "test" {
-	hvn_id         = "test-hvn"
-	cloud_provider = "aws"
-	region         = "us-west-2"
+var vaultCluster = `
+resource "hcp_vault_cluster" "test" {
+	cluster_id            = "test-vault-cluster"
+	hvn_id                = hcp_hvn.test.hvn_id
+	tier				  = "dev"
 }
+`
 
+// sets public_endpoint to true
+var updatedVaultCluster = `
 resource "hcp_vault_cluster" "test" {
 	cluster_id            = "test-vault-cluster"
 	hvn_id                = hcp_hvn.test.hvn_id
-	tier									= "dev"
+	tier				  = "dev"
```

Comment:
Style nit: looks like a mix of tabs and spaces here

Reply:
Gonna make note of this in our test writing readme.